pkg/config/structure/unmarshal.go

```diff
@@ -201,7 +202,9 @@ func copyStruct(target reflect.Value, input nodetreemodel.Node, currPath []strin
 
 			err := copyAny(target.FieldByName(f.Name), input, nextPath, fs)
 			if err != nil {
-				return err
+				warning := fmt.Sprintf("Failed to parse key %q at %v: %v", fieldKey, strings.Join(nextPath, "."), err)
+				log.Warn(warning)
+				continue // don't fail, just warn
```

Comment:
Is this change required, or otherwise covered by any of the tests in this change? It seems at first glance to be unrelated, but maybe I'm missing it.